font-src/glyphs/marks/above.ptl

```diff
@@ -523,6 +535,23 @@ glyph-block Mark-Above : begin
 			flat (m2 - markStress) aboveMarkBot [widths markFine markFine]
 			curl (m2 + markExtend) aboveMarkTop [widths markStress markStress]
 
+	create-glyph 'tripleAcuteAbove' 0x1ACB : glyph-proc
```

Comment:
Please validate it in heavy weight. We may need thinner stroke width for it.

Reply:
Heaviest weight:
![image](https://github.com/be5invis/Iosevka/assets/37010132/ca0f08b4-0978-4bae-9e97-f60955bfaaf0)